engine/src/connector.rs

```diff
@@ -55,7 +54,7 @@ pub struct EthConnector {
     /// It used in the Deposit flow, to verify log entry form incoming proof.
     pub prover_account: AccountId,
     /// It is Eth address, used in the Deposit and Withdraw logic.
-    pub eth_custodian_address: EthAddress,
+    pub eth_custodian_address: Address,
```

Comment:
Couldn't this cause issues?

Reply:
The idea was to remove `EthAdress`. It was `type EthAddress = [u8; 20]`.
But know we have a strong type for that. Type aliasing is sensitive and can't guaranty types consistency.
`Address` type solves that issue.